qutebrowser/browser/commands.py

```diff
@@ -643,14 +643,23 @@ def scroll_perc(self, perc: {'type': float}=None,
 
     @cmdutils.register(instance='command-dispatcher', hide=True,
                        scope='window', count='count')
-    def scroll_page(self, x: {'type': float}, y: {'type': float}, count=1):
+    def scroll_page(self, x: {'type': float}, y: {'type': float},
+            navigate=None, count=1):
```

Comment:
Adding `--next` was my original plan, but then considered that `increment` could also be useful.